eng/scripts/InstallVisualStudio.ps1

```diff
@@ -78,7 +93,11 @@ if (-not $InstallPath) {
 }
 
 if (-not $InstallPath) {
-    $InstallPath = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\$Edition"
+    if ("$Channel" -eq "Preview") {
+        $InstallPath = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\Preview"
```

Comment:
This is going to lead to weird errors if anyone installs multiple preview editions accidentally or on purpose. Suggest
```suggestion
        $InstallPath = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\${Edition}_Pre"
```
or perhaps
```suggestion
        $InstallPath = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019_Pre\$Edition"
```

Reply:
Hi Doug,
thanks for reviewing.

I used `Preview` as that is the default for the Preview version if you use the installer.
If you have more then one version installer, launching regularly the installer you are presented with an error:
> Visual Studio Enterprise 2019 is already installed in this folder. You’ll need to choose another location.
You are then forced to manually change the path:

![image](https://user-images.githubusercontent.com/348793/59567898-7a580500-906b-11e9-8186-9ae95bedbb77.png)

I tried to emulate the behavior, however when the `--installPath` parameter is provided to the installer it just try to modify the installation at that path, even if the installer Edition doesn't match (which feels like a bug in the installer).

About your suggestions, I would prefer the first one as all the edition are installed in the `2019` folder. Probably I would spell out `Preview` instead of `Pre`. What do you think?
Is it worth to add some logic to detect if Preview exists and use this name only as fallback when preview is in use but contains a different edition?

Thank you,
 Alessio

Reply:
This script doesn't need to be perfect because the installer will normally run interactively. So, go with a simple approach that avoids this expected problem i.e. include `$Edition` in the installation location. (The script will update the 2019\Preview folder if the edition installed there matches. But, that "special" default isn't necessary.)

> I would spell out Preview instead of Pre.

My only concern is the length of the path but four characters shouldn't make or break things. If spelling out "Preview" works with the Professional edition (the longest edition name of interest since TestController won't help much), go for it.

Reply:
That's a valid point.
I took your suggestion to stay on the safe side.